index.mjs

```diff
@@ -1,43 +1,5 @@
-'use strict';
```

Comment:
This is required

Reply:
Luckly this file is only available as an `mjs` file in the PR as it stands, which implies strict mode.

There is a good argument for also having a `.js` version linked to in "main" that would require a strict mode directive.

Reply:
The autogenerated index.js file contains this strict mode declaration now.